### ISSUE TITLE
decorators don't work when adding elements through $form->addElements();

### ISSUE THREAD

**Issue by greggberry** [Emagister/zend-form-decorators-bootstrap#82]
elements are wrapped in the default dt dd tags when doing the following:

$text1= new Zend_Form_Element_Text('text1');
$text1->setLabel('Text1');

$text2= new Zend_Form_Element_Text('text2');
$text2->setLabel('Text2');

$this->addElements(array($text1, $text2));

**Comment by nicolashohm** [comment]
i've the same problem :+1:

**Comment by SamHennessy** [comment]
I use  $this->createElement(); to get around this.

E.g.: $dobMonth = $this->createElement('select','dobMonth', array('class' => 'span2'));

**Comment by nicolashohm** [comment]
yep i also use createElement, but i've to change the whole code in my existing project... thats not so nice :wink:
